main.go

```diff
@@ -164,6 +165,8 @@ func main() {
 		go check.IamStatus(bs)
 		// Generate Issuer and Certificate CR
 		go certmanager.DeployCR(bs)
+		// Sync up NSS CR
+		go nss.SyncUpCR(bs)
```

Comment:
I am wondering if we can enable the nss sync up after creating the nss CRs.

Reply:
In the routine, we will wait to sync up until the CRD and the CR itself is ready.
https://github.com/IBM/ibm-common-service-operator/pull/519/files#diff-1b166bbb2715d4ce512c9bea4ae40667c8db17192f7e04d8db3afacfea5755a6R53

I am thinking that if we move this part into `init.go`, it may be not obvious for what additional routine we created in the future code review.